libcst/codegen/gen_matcher_classes.py

```diff
@@ -128,17 +128,6 @@ def leave_ExtSlice(
                     return cst.RemoveFromParent()
         return updated_node
 
-    def leave_Subscript(
```

Comment:
LGTM, other than `leave_Subscript`, I found some other good stuff in libcst/codegen/gen_matcher_classes.py can be moved to libcst/codegen/transforms.py for reusability. E.g. convert ForwardRef as SimpleString.

Reply:
I will move `DoubleQuoteStrings` as well but the ForwardRef handling is entirely related to matchers because of how we codegen using LibCST.